docker/docker-spring-boot-postgres/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.0</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.baeldung.docker</groupId>
+	<artifactId>spring-boot-postgres-docker</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>spring-boot-postgres-docker</name>
```

Comment:
The artifactId and name should match the directory name.

Reply:
See my comments in [BAEL-4652](http://jira.baeldung.com/browse/BAEL-4652?focusedCommentId=152403&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-152403), point 6: "In my code fork, I updated "artifactId" and "name". However, the PR is already closed. Should I create a new PR?"